src/i18n/he.json

```diff
@@ -1,67 +1,69 @@
 {
-  "permanent": "קבוע",
+  "permanent": "Permanent",
   "buttons": {
     "shell": "Toggle shell",
-    "cancel": "Cancel",
-    "close": "Close",
-    "copy": "Copy",
+    "cancel": "ביטול",
+    "close": "סגירה",
+    "copy": "העתקה",
     "copyFile": "Copy file",
     "copyToClipboard": "Copy to clipboard",
-    "create": "Create",
-    "delete": "Delete",
-    "deletePermission": "מחק הרשאה",
-    "removeShare": "Remove Share",
-    "download": "Download",
-    "info": "Info",
-    "more": "More",
-    "move": "Move",
+    "create": "יצירה",
+    "delete": "מחיקה",
+    "deletePermission": "מחיקת הרשאה",
+    "removeShare": "הסרת שיתוף",
+    "download": "הורדה",
+    "info": "פרטים",
+    "more": "עוד",
+    "move": "העבר",
     "moveFile": "Move file",
     "new": "New",
     "next": "Next",
```

Comment:
not used?

Reply:
@omerzamir  I wasn’t sure I translated everything that is still used in the UI so I didn’t delete any translation key for now...

Reply:
I didn't mean you should delete it, I just wondered whether we should've translate it or not.
@omrishtam